crates/driver/src/api/solve/dto/auction.rs

```diff
@@ -87,56 +214,151 @@ enum SellTokenBalance {
 }
 
 #[derive(Debug, Default, Deserialize)]
-#[serde(rename_all = "snake_case")]
+#[serde(rename_all = "lowercase")]
 enum BuyTokenBalance {
     #[default]
     Erc20,
     Internal,
 }
 
-#[serde_as]
-#[derive(Debug, Deserialize)]
-#[serde(rename_all = "camelCase")]
-struct Signature {
-    signing_scheme: SigningScheme,
-    #[serde_as(as = "serialize::Hex")]
-    signature: Vec<u8>,
-}
-
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "lowercase")]
 enum SigningScheme {
     Eip712,
     EthSign,
-    Eip1271,
     PreSign,
+    Eip1271,
 }
 
 #[derive(Debug, Deserialize)]
-#[serde(tag = "class", rename_all = "lowercase")]
-enum OrderClass {
+#[serde(rename_all = "lowercase")]
+enum Class {
     Market,
+    Limit,
     Liquidity,
-    Limit(LimitOrder),
 }
 
 #[serde_as]
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
-struct LimitOrder {
+struct Token {
+    decimals: Option<u8>,
+    symbol: Option<String>,
+    #[serde_as(as = "Option<serialize::U256>")]
+    reference_price: Option<U256>,
     #[serde_as(as = "serialize::U256")]
-    surplus_fee: U256,
+    available_balance: U256,
+    trusted: bool,
 }
 
-#[derive(Debug, Default, Deserialize)]
-struct Interactions {
-    pre: Vec<InteractionData>,
+#[derive(Debug, Deserialize)]
+#[serde(tag = "kind", rename_all = "lowercase")]
+enum Liquidity {
```

Comment:
This shouldn't be here though right?

There is no liquidity DTO from `autopilot` -> `driver`.